### PR TITLE
Admin Portal performance improvements

### PR DESCRIPTION
**Pull request by kavishkafernando**
Admin Portal performance improvements

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/ProtectedApp.jsx [hunk 1]
```diff
@@ -182,6 +182,7 @@ class Protected extends Component {
             clientId: Utils.getCookieWithoutEnvironment(User.CONST.ADMIN_CLIENT_ID),
             sessionStateCookie: Utils.getCookieWithoutEnvironment(User.CONST.ADMIN_SESSION_STATE),
             mobileOpen: false,
+            isTenant: false,
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/ProtectedApp.jsx:185]: Shall we give a proper name to this variable, `isTenant` does not mean anything here right? shouldn't it be `isSuperTenant` ?

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/ProtectedApp.jsx [hunk 2]
```diff
@@ -212,6 +213,18 @@ class Protected extends Component {
...
+                .then((result) => {
+                    const { tenantDomain } = result.body;
+                    if (tenantDomain === 'carbon.super') {
+                        this.setState({ isTenant: false });
+                    } else {
+                        this.setState({ isTenant: true });
+                    }
+                })
+                .catch((error) => {
+                    console.log(error);
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/ProtectedApp.jsx:226]: Shall we alert the user about the error ? and log

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/webpack.config.js [hunk 3]
```diff
@@ -32,6 +32,10 @@ const config = {
         net: 'empty', // To fix joi issue: https://github.com/hapijs/joi/issues/665#issuecomment-113713020
     },
     watch: false,
+    watchOptions: {
```
Comment (tmkasun) [inline features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/webpack.config.js:35]: 😑

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/ProtectedApp.jsx (hunk 1 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/ProtectedApp.jsx:185]: The API call is to check the tenant information that's why i named that variable isTenant.

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/webpack.config.js (hunk 3 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/webpack.config.js]: Removed with `f01475d`

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/ProtectedApp.jsx (hunk 2 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/ProtectedApp.jsx:228]: Fixed with `f01475d`

File: features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/ProtectedApp.jsx (hunk 1 above)
Comment (kavishkafernando) [reply features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/ProtectedApp.jsx]: Fixed with `f01475d`

**Pull request merged**